### DISCUSSION TITLE
How to tell Symbolic Regression to create trees without using constants?

### DISCUSSION THREAD

**Discussion by vinhpb** [MilesCranmer/PySR#417] [Q&A] [answered]
Hi,

I wonder how can we tell SymbolicRegression.jl to not use any constants (e.g., 2.0, 15.6) when it creates trees. I am using SRRegressor via MLJ interface. Thank you.

Regards

**Comment by MilesCranmer** [comment, marked as answer]
You can set the complexity of constants to a very large number. Like `complexity_of_constants=100`. Then it will avoid them.

**Reply by vinhpb** [reply]
That works. Thanks a lot! 👍
